src/Indice.Services/ViberServiceApifon.cs

```diff
@@ -0,0 +1,134 @@
+﻿using System.Net.Http.Headers;
+using System.Net.Mime;
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Indice.Globalization;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace Indice.Services;
+
+/// <summary>Viber/SMS service implementation using the Apifon IM service gateway.</summary>
+/// https://docs.apifon.com/apireference.html#im-gateway-rest-api
+public class ViberServiceApifon : ISmsService
```

Comment:
Same as with the IServiceCollection extension method.

rename class `ViberServiceApifon`to  `SmsServiceApifonIM`